API.md

```diff
@@ -4581,6 +4581,12 @@ Request information:
 
 Note that the `request.info` object is not meant to be modified.
 
+#### <a name="request.isInjected" /> `request.isInjected`
+
+Access: read only.
```

Comment:
Is this actually enforced at all? If not, we could use a getter like some other properties in `Request`.

Reply:
Will implement in next change-set + unit test.